### PR TITLE
Autogenerate release blog pages

### PR DESCRIPTION
**Pull request by Grifs**
Add script to convert CHANGELOG.md to release blog posts
Add CHANGELOG.md in reference
Run script & update existing pages + some new pages

### CODE REVIEW

**Review by rcannood — APPROVED** [review]
LGTM!

Is the `reference/CHANGELOG.md` not redundant?

File: bin/scripts/generate_version_blog_pages.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+def bump_markdown_headers(lines: list[str]) -> list[str]:
+  """ map headers and bump up one level """
+  out = map(bump_md_line, lines)
+  return list(out)
+
+def handle_title(header: str) -> (str, str):
+  """ Split the title and create a header section """
+  matches = re.search(r"^# Viash (\d+[\.\d+]+) \(([\dy]{4}-[\dM]{2}-[\dd]{2})\): (.*)$", header)
+
+  version = matches.group(1)
```
Comment (DriesSchaumont) [inline bin/scripts/generate_version_blog_pages.py:46]: Just a quick tip:

```python3
version, date, subtitle = matches.group(1,2,3)
```

**Review by DriesSchaumont — APPROVED** [review]
Quick tip for future reference, but otherwise OK

File: bin/scripts/generate_version_blog_pages.py (hunk 1 above)
Comment (Grifs) [reply bin/scripts/generate_version_blog_pages.py:46]: Aha! Good to know, thanks!

**Comment by Grifs** [comment]
> Is the `reference/CHANGELOG.md` not redundant?

Redundant or rather completely in the wrong folder?
I moved the changelog to `bin/data`, maybe in the future the `*_schema_export.json` files can also move there?

**Pull request merged**